gdcdictionary/python/__init__.py

```diff
@@ -65,7 +66,7 @@ def load_yaml(self, name):
         """Return contents of yaml file as dict"""
         # For DAT-1064 Bomb out hard if unicode is in a schema file
         # But allow unicode through the terms and definitions
-        with open(name, 'r') as f:
+        with codecs.open(name, 'r', encoding="utf-8") as f:
```

Comment:
The problem is in _terms.yaml.
```
+route_of_administration:
+      <U+200B>The pathway by which a substance is administered in order to reach the site
+      of action in the body.
+        <U+200B>Route of Administration
+      cde_id: C38114<U+200B>
+      cde_version: 21.04d
+      term_url: "https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&ns=ncit&code=C38114"
```
There are now zero-width whitespace characters (U+200B).  They should be deleted.

Reply:
Nice catch. I wonder if a word processor inserted these characters.

Reply:
I tried to dig into the source behind the comment about allowing unicode in terms and definitions but found nothing